ui/imports/shared/popups/send/SendModal.qml

```diff
@@ -24,9 +24,14 @@ import "./views"
 StatusDialog {
     id: popup
 
-    property string preSelectedRecipient
+    // expected content depends on the preSelectedRecipientType value.
+    // If type Address this must be a string else it expects an object. See RecipientView.selectedRecipientType
+    property var preSelectedRecipient
+    property int preSelectedRecipientType: TabAddressSelectorView.Type.Address
     property string preDefinedAmountToSend
+    // requires to have assigned an item from assets model
     property var preSelectedHolding
+    // token symbol
```

Comment:
leftover??

Reply:
No, that should hold the token symbol in case of `preSelectedRecipientType: TabAddressSelectorView.Type.Address`. I learned this from @Khushboo-dev-cpp